index.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+</head>
+
+<body>
+<div class="resume-container">
+    <header>
+        <h1>CHARLES MCTURLAND</h1>
+        <p class="title">SOFTWARE ENGINEER</p>
+    </header>
+    <div class="content">
```

Comment:
I think you can use a semantic tag here rather than a div

Reply:
![div](https://github.com/user-attachments/assets/74584dd1-82e6-4efb-8b4c-c8b87c443b48)

I tried to use something else, but it ruins the whole lay out. I also used flex on that div to keep everything inside intact.

Reply:
I don't think so. I've just tried to replace this `div` with the `main` tag and it works like a charm